package/tests/conftest.py

```diff
@@ -120,7 +119,7 @@ def example_transcoded_catalog():
     yield DataCatalog(
         data_sets={
             "model_inputs@pandas": ParquetDataSet(filepath="model_inputs.parquet"),
-            "model_inputs@spark": SparkDataSet(filepath="model_inputs.csv"),
+            "model_inputs@pandas2": CSVDataSet(filepath="model_inputs.csv"),
```

Comment:
Getting rid of the dependence on spark is a great idea, I really like this. Is there anything else in our config.yml that mentions spark that we can get rid of also?

Reply:
there is in js tests but it is totally unrelated to this. It is just front-end testing if things work and doesn't have any python dependency